src/components/Input/Input.d.ts

```diff
@@ -74,13 +74,6 @@ export interface InputProps {
 	 */
     className?: string;
 
-    /**
-	 * Type of the input element
-	 *
-	 * @default 'text'
-	 */
-    type?: string;
```

Comment:
why this change?

Reply:
`type` is included in the `FormControlProps` extended above